.rubocop.yml

```diff
@@ -53,3 +53,9 @@ Style/MultilineBlockChain:
 # Allow Perl-style references to regex matches
 Style/PerlBackrefs:
   Enabled: false
+
+Style/Documentation:
+  Exclude:
+    - 'lib/reek/ast/sexp_extensions/send.rb'
```

Comment:
Those changes are in here because rubocop starts to [complain](https://github.com/troessner/reek/pull/746/files#diff-40de68a2762069e9358b2758b12202e2R49) all of a sudden. I haven't investigated why this is exactly but I consider it a rubocop bug which should not block this pull request.
I'd rather address this in a follow issue / PR.